spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -298,9 +308,353 @@
         end
 
         context 'selfie with error is uploaded' do
+          before do
+            allow(IdentityConfig.store).to receive(:doc_auth_max_attempts).and_return(99)
+
+            allow_any_instance_of(FederatedProtocols::Oidc).
+              to receive(:biometric_comparison_required?).
+              and_return(true)
+            perform_in_browser(:mobile) do
+              visit_idp_from_oidc_sp_with_ial2(biometric_comparison_required: true)
+              sign_in_and_2fa_user(@user)
+              complete_doc_auth_steps_before_document_capture_step
+            end
+          end
+
+          it 'shows the correct error message for the given error' do
+            # when the only error is a doc auth error
+
+            perform_in_browser(:mobile) do
+              attach_images(
+                Rails.root.join(
+                  'spec', 'fixtures',
+                  'ial2_test_credential_doc_auth_fail_selfie_pass.yml'
+                ),
+              )
+              attach_selfie(
+                Rails.root.join(
+                  'spec', 'fixtures',
+                  'ial2_test_credential_doc_auth_fail_selfie_pass.yml'
+                ),
+              )
+
+              submit_images
+
+              h1_error_message = strip_tags(t('errors.doc_auth.rate_limited_heading'))
+              expect(page).to have_content(h1_error_message)
+
+              body_error_message = strip_tags(t('doc_auth.errors.dpi.top_msg'))
+              expect(page).to have_content(body_error_message)
+
+              click_try_again
+              expect(page).to have_current_path(idv_document_capture_path)
+
+              inline_error_message = strip_tags(t('doc_auth.errors.dpi.failed_short'))
```

Comment:
In the figma I'm seeing that when it's a doc auth fail we should be using `top_msg` rather than `failed_short`?

Reply:
I think you're right. This is a case of what I mentioned where I think the inline error message is wrong but I'm not sure if fixing it is in the scope of this PR. I'm wondering if I should add this onto the inline error ticket, or make a new ticket to fix these later.